src/components/Header/Nav/SubNav/SubNavList/SubNavListItem/SubNavButton.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import styled from 'preact-emotion';
+import { connect } from 'unistore/preact';
+
+import { headline } from 'pasteup/fonts';
+import { pillars } from 'pasteup/palette';
+import { desktop } from 'pasteup/breakpoints';
+
+const SubNavButton = styled('button')(
+    ({ pillar, isLastIndex, showSecondaryNav }) => {
+        const styles = {
```

Comment:
jsut out of interest, why define `styles` here, rather than just return the object directly?

Reply:
yeah you could just do`const SubNavButton = styled('button')(({ pillar, isLastIndex, showSecondaryNav }) => ({...styles}))` here. I probably had more complex `if` conditions in the function block at some point that've gone. This can be refactored.